### PR TITLE
PHP: complete some todo

### PR DESCRIPTION
**Pull request by thinkerou**

### CODE REVIEW

**Comment by thelinuxfoundation** [comment]
Thank you for your pull request. Before we can look at your contribution, we need to ensure all contributors are covered by a Contributor License Agreement.

After the following items are addressed, please respond with a new comment here, and the automated system will re-verify.

* One or more commits are not associated with a GitHub user. The email used in the Git commit must match a verified email address in a GitHub user profile. For more information, please see https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/

_Regards,
CLA GitHub bot_

**Comment by thinkerou** [comment]
@ZhouyihaiDing Hi, please review the pr, thanks a lot.

**Comment by ZhouyihaiDing** [comment]
Interop Cloud-to-Prod Tests (internal CI) - #15231

File: src/php/ext/grpc/php7_wrapper.h [hunk 1]
```diff
@@ -154,6 +156,11 @@ static inline int php_grpc_zend_hash_find(HashTable *ht, char *key, int len,
   add_property_string(arg, name, context)
 #define php_grpc_add_property_stringl(res, name, str, len, b) \
   add_property_stringl(res, name, str, len)
+#define php_grpc_add_property_zval(res, name, val) do { \
+  zval tmp; \
+  tmp = *val; \
+  add_property_zval(res, name, &tmp); \
+  } while(0)
```
Comment (ZhouyihaiDing) [inline src/php/ext/grpc/php7_wrapper.h]: Why you need a while(0) here

Comment (thinkerou) [reply src/php/ext/grpc/php7_wrapper.h]: Here use `do {} while` but loop `zero` time.

Comment (ZhouyihaiDing) [reply src/php/ext/grpc/php7_wrapper.h]: What is the `statements` mean?
Is it necessary to have this `zero` time loop?

Comment (ZhouyihaiDing) [reply src/php/ext/grpc/php7_wrapper.h]: I got it. Thanks!

**Pull request merged**